node-api-engine-bindings.md

```diff
@@ -67,6 +67,7 @@ language to implement native addons for Node.js through Node-API:
 |[electron](https://github.com/electron/electron)| C++|
 |[iotjs](https://github.com/jerryscript-project/iotjs)| C|
 |[veil](https://github.com/lightsourceengine/veil)| C|
+|[emnapi](https://github.com/toyobayashi/emnapi)| C / JavaScript|
```

Comment:
Would you mind sorting these entries alphabetically?

Reply:
done